keyboards/akko/top40/keymaps/vial/config.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Oleg Titov <oleg.titov@gmail.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Unique keyboard id
+#define VIAL_KEYBOARD_UID {0x5D, 0x72, 0x7C, 0xB4, 0xC9, 0x7C, 0xF2, 0x60}
+
+// Unlock with Escape+Enter
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 1 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 11 }
+
+// Enable all available RGB Matrix effects
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
+
+// Firmware size optimization
+// disable Lock Key support
+#undef LOCKING_SUPPORT_ENABLE
+#undef LOCKING_RESYNC_ENABLE
```

Comment:
```suggestion

```
Any reason this was added?

Reply:
That removes support for nonpresent [Cherry MX Lock switch](https://docs.qmk.fm/#/squeezing_avr?id=configh-settings). I assume that could stay, what do you think?

Reply:
Ok, let's remove it too just in case some one will get these switches.